includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -455,6 +455,12 @@ private function update_is_upe_enabled( WP_REST_Request $request ) {
 		}
 
 		$settings = WC_Stripe_Helper::get_stripe_settings();
+
+		// If the new UPE is enabled, we need to remove the flag to ensure legacy SEPA tokens are updated flag.
+		if ( $is_upe_enabled && ! WC_Stripe_Feature_Flags::is_upe_checkout_enabled() ) {
+			delete_option( 'woocommerce_stripe_subscriptions_legacy_sepa_tokens_updated' );
```

Comment:
When a admin user enables the new checkout experience we would scheduled the migration jobs and then set an option to make sure we don't reschedule them. 

However, if the merchant noticed that their subscriptions were manual and in a panic went back and switched on the legacy checkout, we didn't delete the option. This means that if the merchant in the future switched legacy back off again, any SEPA subscriptions wouldn't be updated because the original option locked was still in place. 

This change makes sure that when a with legacy checkout enabled, disables it, we delete the option which should restart the subscription migrations.